tests/backup/backup_helper.go

```diff
@@ -1696,7 +1694,7 @@ func DeletePodWithWithoutLabelInNamespace(namespace string, label map[string]str
 }
 
 // backupSuccessCheck inspects backup task
-func backupSuccessCheck(backupName string, orgID string, retryDuration time.Duration, retryInterval time.Duration, ctx context.Context) error {
+func backupSuccessCheck(backupName string, orgID string, retryDuration time.Duration, retryInterval time.Duration, ctx context.Context, SizeCheck bool) error {
```

Comment:
Wouldn't it be better if we add size check to `ValidateBackup` instead of adding it here?

Reply:
The reason of going for with BackupSuccessCheck is because we can do this check immediately after the backup is done.

Reply:
What is the difference between checking it here and `ValidateBackup`?
The objective of the function `backupSuccessCheck` is to wait for the response from px-backup api server and come out once we get a success message.
We have grouped all the validation in `ValidateBackup`

Reply:
Can we also not add a new parameter and change the signature of the function?
We inspect the backup object and check the number of volumes which are getting backed up. If the number of volumes is more than 1 then we will do a size check.

Reply:
Based on alot of discussion let's go ahead with this change for now.